tests/em_test_helper.rb

```diff
@@ -1,8 +1,11 @@
+require 'em/pure_ruby' if ENV['EM_PURE_RUBY']
```

Comment:
Given this environment variable, we could trigger it either with the test_pure top-level Rake task, or with a `.travis.yml` flag -- in the latter case, the entire Ruby version and platform matrix could be executed twice to ensure wide compatibility.